front/lib/extract_events.ts

```diff
@@ -140,16 +142,21 @@ async function _processExtractEvent(data: {
 
   const results = await _runExtractEventApp(auth, inputsForApp);
   results.map(async (result: string) => {
-    // @todo be smarter
-    // check that this event is not already in the database
-    // check the properties match what's expected (json is typed on model)
-    // handle errors
     const properties = JSON.parse(result);
+    if (!properties.marker) {
```

Comment:
I might be missing something but why rely on the model to get the marker?

Reply:
Is the marker the full one? like `product_idea:foo` ?

Reply:
Looking at the app the function prompt is:
```
marker:
{ type, enum, description }
type:
string
enum:
[ 1 items ]
description:
This field must be returned and filled with idea:8
```

So it seems like it's just an echo? What's the purpose?

Reply:
As per IRL the marker should be handled symbolically not neurally. I presume you can remove it from the app output / function call and just recover it from the args you're passing to the app directly?

We can also probably kill `marker_log` as well?